modules/yaml/yaml.go

```diff
@@ -35,13 +35,22 @@ func renderHorizontalHtmlTable(m yaml.MapSlice) string {
 		key := mi.Key
 		value := mi.Value
 
-		if key != nil && reflect.TypeOf(key).String() == "yaml.MapSlice" {
+		switch reflect.TypeOf(key).String() {
```

Comment:
This is not a [type switch](https://golang.org/doc/effective_go.html#type_switch). I'd prefer a type switch to a regular switch (which is basically the same as what we originally had) to avoid having magic strings like `"[]interface {}"` floating around (which breaks if it's missing an easy-to-forget space).

Ditto elsewhere.